elements/jsonform/src/main.js

```diff
@@ -88,34 +88,31 @@ export class EOxJSONForm extends LitElement {
   /**
    * Get The JSON schema for rendering the form
```

Comment:
Could you update this description? Since now it does not only get, but also set. In the Storybook Docs it could be a bit confusing otherwise:

![image](https://github.com/EOX-A/EOxElements/assets/26576876/fd2615bc-288b-4dde-8cb9-8743961db875)